packages/block-library/src/blocks/card/init.php

```diff
@@ -28,6 +28,10 @@ function novablocks_render_card_block( $attributes, $content ) {
 			'novablocks-block__content'
```

Comment:
It may not be an issue, but the `preview` version inside the block editor also has the `novablocks-card--fixed-media-aspect-ratio` classname which seems to be missing here.